great_expectations/data_context/store/json_site_store.py

```diff
@@ -0,0 +1,158 @@
+        icon:
+        short_description: Filesystem
+        description: Use a locally-mounted filesystem to store validation results.
+        how_to_guide_url: https://docs.greatexpectations.io/en/latest/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_on_a_filesystem.html
+        maturity: Production
+        maturity_details:
+            api_stability: Stable
+            implementation_completeness: Complete
+            unit_test_coverage: Complete
+            integration_infrastructure_test_coverage: N/A
+            documentation_completeness: Complete
+            bug_risk: Low
+
+        id: validations_store_s3
+        title: Validations Store - S3
+        icon:
+        short_description: S3
+        description: Use an Amazon Web Services S3 bucket to store validation results.
+        how_to_guide_url: https://docs.greatexpectations.io/en/latest/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_s3.html
+        maturity: Beta
+        maturity_details:
+            api_stability: Stable
+            implementation_completeness: Complete
+            unit_test_coverage: Complete
+            integration_infrastructure_test_coverage: Minimal
+            documentation_completeness: Complete
+            bug_risk: Low
+
+        id: validations_store_gcs
+        title: Validations Store - GCS
+        icon:
+        short_description:
+        description: # What it does  <br /> Store validation results in a Google Cloud Storage bucket. You may optionally specify a key to use. <br /> <br /> See the GCS Store backend [module docs](https://docs.greatexpectations.io/en/latest/autoapi/great_expectations/data_context/store/tuple_store_backend/index.html#great_expectations.data_context.store.tuple_store_backend.TupleGCSStoreBackend) for more information."
+        how_to_guide_url: https://docs.greatexpectations.io/en/latest/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_gcs.html
+        maturity: Beta
+        maturity_details:
+            api_stability: Stable
+            implementation_completeness: Complete
+            unit_test_coverage: Complete
+            integration_infrastructure_test_coverage: Minimal
+            documentation_completeness: Partial
+            bug_risk: Low
+
+        id: validations_store_azure_blob_storage
+        title: Validations Store - Azure
+        icon:
+        short_description: Azure Blob Storage
+        description: Use Microsoft Azure Blob Storage to store validation results.
+        how_to_guide_url:
+        maturity: N/A
+        maturity_details:
+            api_stability: Stable
+            implementation_completeness: Minimal
+            unit_test_coverage: Minimal
+            integration_infrastructure_test_coverage: Minimal
+            documentation_completeness: Minimal
+            bug_risk: Moderate
+
+    --ge-feature-maturity-info--
```

Comment:
We should update this docstring. No need to replicate the feature maturity structure for now - a sentence or two should do.

Reply:
updated